source/MaterialXRenderGlsl/GlslRenderer.cpp

```diff
@@ -190,13 +207,15 @@ void GlslRenderer::updateViewInformation(const Vector3& eye,
                                            float farDist,
                                            float objectScale)
 {
+    float aspectRatio = float(_width) / float(_height);
```

Comment:
Small logic change to compute a fov which is non-square. The geometry fit is just a heuristic to start with, but make it still fit inside.